packages/block-editor/src/components/list-view/appender.js

```diff
@@ -58,11 +74,24 @@ export const Appender = forwardRef(
 			);
 		}, [ insertedBlockTitle ] );
 
+		const orderInitialBlockItems = useCallback( ( items ) => {
+			items.sort( ( { id: aName }, { id: bName } ) => {
+				// Sort block items according to `prioritizedInserterBlocks`.
+				let aIndex = prioritizedInserterBlocks.indexOf( aName );
+				let bIndex = prioritizedInserterBlocks.indexOf( bName );
+				// All other block items should come after that.
+				if ( aIndex < 0 ) aIndex = prioritizedInserterBlocks.length;
+				if ( bIndex < 0 ) bIndex = prioritizedInserterBlocks.length;
+				return aIndex - bIndex;
+			} );
+			return items;
+		}, [] );
+
 		if ( hideInserter ) {
 			return null;
 		}
-
 		const descriptionId = `list-view-appender__${ instanceId }`;
+		const { PrivateInserter } = unlock( blockEditorPrivateApis );
```

Comment:
Why is this being unlocked via the block editor private APIs? We're currently in the block editor, so I thought we could import directly from that component, as it's a relative import rather than reaching out to a separate package.